ruby-macho.gemspec

```diff
@@ -13,4 +13,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.6"
   s.homepage = "https://github.com/Homebrew/ruby-macho"
   s.license = "MIT"
+  s.metadata["rubygems_mfa_required"] = "true"
```

Comment:
not sure what aded this - assume rubocop? (not mad at it)

Reply:
Huh, interesting. Well, my RubyGems account already has MFA, so that shouldn't be a problem.